keymaps/advanced-open-file.cson

```diff
@@ -27,6 +27,7 @@
   'ctrl-k': 'advanced-open-file:move-cursor-down'
 
 '.platform-darwin .advanced-open-file atom-text-editor':
+  'ctrl-l': 'advanced-open-file:delete-a-path-component'
```

Comment:
This is totally minor, but `delete-path-component` matches the language of the other commands better. Honestly, though, that's just me being nitpicky. :P